app/src/main/res/values/strings.xml

```diff
@@ -697,6 +697,16 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
 
     <string name="quest_bridge_structure_title">What’s the structure of this bridge?</string>
 
+    <string name="quest_building_entrace_title">What kind of building entrance is this?</string>
+    <string name="quest_building_entrance_main">Main entrance of a public building</string>
+    <string name="quest_building_entrance_staircase">Staircase entrance (e.g. of apartment building)</string>
+    <string name="quest_building_entrance_service">Service entrance</string>
+    <string name="quest_building_entrance_exit">Exit only</string>
+    <string name="quest_building_entrance_emergency_exit">Emergency exit only</string>
+    <string name="quest_building_entrance_shop">Shop entrance (storefront entrance), not a main building entrance</string>
```

Comment:
```suggestion
    <string name="quest_building_entrance_shop">Storefront entrance</string>
```

Too long otherwise IMO

Reply:
`Storefront entrance (not main entrance)` ?

Reply:
If it was a main entrance, the user would have chosen that already because it is displayed further up, no? Also, if there is a single shop in a building and that building has one entrance, "storefront entrance (not main entrance)" would be confusing